packages/ui-components/src/components/Button.tsx

```diff
@@ -45,16 +45,27 @@ export interface ButtonProps extends MuiButtonProps {
   loadingText?: string;
 }
 
-export const Button = ({
-  children,
-  isLoading = false,
-  loadingText = 'Loading',
-  disabled = false,
-  ...props
-}: OverrideableComponentProps<ButtonProps>) => (
-  <StyledButton variant="contained" color="primary" {...props} disabled={isLoading || disabled}>
-    {isLoading ? `${loadingText}...` : children}
-  </StyledButton>
+export const Button = forwardRef(
+  (props: OverrideableComponentProps<ButtonProps>, ref: Ref<any>) => {
```

Comment:
thanks for fixing this up. That console error has been staring at me for a while now